packages/create-react-native-library/src/inform.ts

```diff
@@ -96,6 +96,16 @@ export async function printNextSteps(
         )
         .join('\n')}
 
+      ${
+        config.project.nitro
+          ? kleur.blue(
+              `Read the ${kleur.bold(
+                'Nitro Modules'
+              )} docs at ${kleur.underline('https://nitro.margelo.com')}
+              `
+            )
+          : ''
+      }
       ${kleur.yellow(
         `See ${kleur.bold('CONTRIBUTING.md')} for more details. Good luck!`
       )}
```

Comment:
Imo it's a bit awkward:

```
Read [..] at [...]
See [..] for more details
```

Reads like the the see for more details is talking about the link above. I'd move the nitro link to below, and add a new line between them. Or remove the link altogether since it's in CONTRIBUTING.md anyway - and people don't read that. So not sure if 2 calls to action is better.